### PR TITLE
Trend micro dda bug

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-20612

## Description
Fixed an issue where the migration to python3 resulted in an error.

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@yuvalbenshalom Doc review completed.

File: Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA.py [hunk 1]
```diff
@@ -72,12 +72,12 @@
...
             temp += headers[key]
     else:
-        for key, value in headers.iteritems():
+        for key, value in headers.items():
             if ('X-DTAS-' in key and 'X-DTAS-Checksum' not in key and 'X-DTAS-ChecksumCalculatingOrder' not in key):
                 temp += value
 
     temp += body
-    return hashlib.sha1(temp)  # nosec
+    return hashlib.sha1(temp.encode('utf-8'))  # nosec
```
[the remark this answers is not in the document]
Comment (yuvalbenshalom) [reply Packs/TrendMicroDDA/Integrations/TrendMicroDDA/TrendMicroDDA.py:80]: this only used for checksum

File: Packs/TrendMicroDDA/ReleaseNotes/1_0_12.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Trend Micro Deep Discovery Analyzer (Beta)
+- Fixed an issue caused by moving the integration to Python 3.
```
Comment (yuvalbenshalom) [inline Packs/TrendMicroDDA/ReleaseNotes/1_0_12.md:4]:
```suggestion
- Fixed an issue where the integration failed to run.
```

**Review by yuvalbenshalom — APPROVED** [review]
see my RN comment

**Pull request merged**